awx_collection/tools/README.md

```diff
@@ -0,0 +1,61 @@
+## Collection tools
+
+Tools used for building, maintaining, and testing the collection.
+
+### Template Galaxy
+
+The `template_galaxy.yml` playbook ran as a pre-requisite for building the collection.
+
+```
+make awx_collection_build
+```
+
+### Generate
+
+This will template resource modules (like `group`, for groups in inventory) from a boilerplate template.
+It is intended as a tool for writing new modules or enforcing consistency.
+
+### Integration Testing
+
+These instructions assume you have ansible-core and the collection installed.
+To install the collection in-place (to pick up any local changes to source)
+the `make symlink_collection` will put in place symlinks.
```

Comment:
this reads funny to me. will put in place symlinks?

Reply:
I'll say "will symlink the folder"